src/status_im/ui2/screens/chat/composer/input.cljs

```diff
@@ -210,7 +210,7 @@
     (if platform/android?
       [selectable-text-input chat-id props children]
       [rn/text-input props
-       children])))
+       [children]])))
```

Comment:
unfortunately, it's not that simple, there is differently something wrong, it doesn't play well with mentions, also we shouldn't use default-value, because we set default value as children

Reply:
yes, i tested on ios as well

Reply:
for some reason it starts to duplicate text, when you typing or when you select mention